### PR TITLE
Flex-198: Extended flex status command to show deployment information

### PR DESCRIPTION
**Pull request by galexandrov**
The deployment status will be shown when `flex-operator` feature falg is enabled for application

Flex status command output

```
key                value                                
-----------------  -------------------------------------
status             ONLINE                               
version            1.4.2                                
runtime            FLEX-RUNTIME-NODE8                   
id                 123456                               
requestedAt        replaced_value
deployerEmail      davy.jones@mail.com                  
deployerName       Davy Jones                           
deploymentStatus   COMPLETED                            
deploymentVersion  1.4.2                                
deploymentRuntime  FLEX-RUNTIME-NODE10  
```

### CODE REVIEW

File: test/fixtures/datalinks-status-response.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      "deployUserInfo": {
+        "firstName": "Davy",
+        "lastName": "Jones",
+        "email": "davy.jones@mail.com"
+      },
+      "version": "1.4.2",
+      "runtime": "flex-runtime-node8",
+      "deployment": {
+        "status": "COMPLETED",
+        "version": "1.4.2",
```
Comment (tsvetomir-nedyalkov) [inline test/fixtures/datalinks-status-response.json]: It is good 'version' to be different from 'deployment.version' in order to validate that the CLI reads and displays the correct fields from MAPI

File: test/fixtures/datalinks-status-response.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      "requestedAt": "2017-11-06T03:42:31.970Z",
+      "deployUserInfo": {
+        "firstName": "Davy",
+        "lastName": "Jones",
+        "email": "davy.jones@mail.com"
+      },
+      "version": "1.4.2",
+      "deployment": {
+        "status": "COMPLETED",
+        "version": "1.4.2",
```
Comment (tsvetomir-nedyalkov) [inline test/fixtures/datalinks-status-response.json]: Same as above

File: lib/Constants.js [hunk 3]
```diff
@@ -227,6 +227,12 @@ Constants.ServiceStatus = {
   ERROR: 'ERROR'
 };
 
+Constants.DeploymentStatus = {
```
Comment (tsvetomir-nedyalkov) [inline lib/Constants.js]: It seems that these Constant is not used anywhere...

Comment (galexandrov) [reply lib/Constants.js]: https://github.com/Kinvey/kinvey-cli/blob/flex-198-operator-integration/lib/flex/FlexService.js#L34

**Pull request merged**